plot_continuous_monitor.py

```diff
@@ -226,9 +153,171 @@ def get_util_for_disk(disk_utils, disk):
       return disk_util[disk]
   return None
 
+def gnuplot_data_line(data_hash):
+    """ Helper function to order continuous montior data for gnuplot to use """
+    return [data_hash['time'],
+            data_hash['xvdf utilization'],
+            data_hash['xvdb utilization'],
+            data_hash['cpu utilization'],
+            data_hash['bytes received'],
+            data_hash['bytes transmitted'],
+            data_hash['running compute monotasks'],
+            data_hash['running monotasks'],
+            data_hash['gc fraction'],
+            data_hash['outstanding network bytes'],
+            data_hash['macrotasks in network'],
+            data_hash['macrotasks in compute'],
+            data_hash['cpu system'],
+            data_hash['macrotasks in disk'],
+            data_hash['xvdf read throughput'],
+            data_hash['xvdf write throughput'],
+            data_hash['xvdb read throughput'],
+            data_hash['xvdb write throughput'],
+            data_hash['xvdf running disk monotasks'],
+            data_hash['xvdb running disk monotasks'],
+            data_hash['free heap memory'],
+            data_hash['free off heap memory'],
+            data_hash['local running macrotasks']]
+
+
+def plot_gnuplot(file_prefix, open_graphs):
+  """
+  Creates modified copies of gnuplot files, and associated pdf graphs for several
```

Comment:
What do you mean modified copies? What about "Creates gnuplot files that can be used to generate plots with data for various continuous monitor attributes, and uses those files to generate PDFs of each plot."